packages/drupal-kit/src/lib/index.ts

```diff
@@ -0,0 +1,4 @@
+export { default as defaultFetch } from './defaultFetch';
```

Comment:
@backlineint I flattened out the folders a bit, I wasn't sure the reason for `fetch` having its own directory but we can bring that back if needed.